app/lib/shared/task_scheduler.dart

```diff
@@ -132,7 +132,7 @@ class PrioritizedStreamIterator<T> implements StreamIterator<T> {
 
     // Listen on the streams and put items into their own queues.
     for (int i = 0; i < sources.length; i++) {
-      final Stream source = sources[i];
+      final source = sources[i];
```

Comment:
Personally I don't think removing types helps readability of the code. I understand that there are lots of cases where a type can be inferred, but that doesn't mean that the code is easier to read/understand.

So why not change this to `Stream<T>` instead?

Reply:
> So why not change this to Stream<T> instead?

Just a matter of style. I don't mind inference. Less typing. Can go either way.